### ISSUE TITLE
Core Eval (minor?) Issues

### ISSUE THREAD

**Issue by hijohnnylin** [adamkarvonen/SAEBench#37]
Two potential issues with core evals, none of which require us to rerun (I believe).
 
- it seems not possible to disable any of the flags eg `compute_featurewise_density_statistics`, since most times we're calling through `multiple_evals` which calls `get_multiple_evals_everything_config` and it sets everything to true.

- we should have separate model dtype and sae dtype. I believe for Gemma Scope the SAEs were float32 and the model is bfloat16

**Comment by adamkarvonen** [comment]
Agreed on #1. It would be nice to make it easy to disable those flags.

For dtype, we have been matching model dtype and sae dtype (and thus, maybe we should rename llm_dtype to dtype, as this is confusing). In section 4.7 of the GemmaScope paper, they said using bfloat16 should have a negligible difference for inference. This also avoids potential headaches with dtype mismatch.

**Comment by hijohnnylin** [comment]
awesome, thanks!

for dtype, thanks for the pointer to the gemma scope paper, this is great. i actually prefer not changing the eval config schema for now since we'd have to go and update the config metadata on the other core evals we already ran.

**Comment by adamkarvonen** [comment]
This was resolved in this commit: https://github.com/adamkarvonen/SAEBench/commit/f6430afc9152bc7d8ab894fc1e047fc07e6c6427
